releases/release-1.16/release_team.md

```diff
@@ -4,7 +4,7 @@
 | Enhancements | Kendrick Coleman ([@kacole2](https://github.com/kacole2) / Slack: `@kacole2`) |  |
 | CI Signal | Jorge Alarcon ([@alejandrox1](https://github.com/alejandrox1) /Slack: `@alejandrox1`) | Alena Varkockova ([@alenkacz](https://github.com/alenkacz) / Slack: `@alenkacz`), Jim Angel ([@jimangel](https://github.com/jimangel) / Slack: `@jimangel`), Nicholas Lane ([@soggiest](https://github.com/soggiest) / Slack: `@soggy`), Veronica Lopez ([@verolop](https://github.com/verolop) / Slack: `@Verónica López`) |
 | Bug Triage | Marko Mudrinic ([@xmudrii](https://github.com/xmudrii) / Slack: `@xmudrii`) | |
-| Branch Manager | Yang Li ([@idealhack](https://github.com/idealhack) / Slack `@idealhack`) | |
+| Branch Manager | Yang Li ([@idealhack](https://github.com/idealhack) / Slack `@idealhack`) | see [Release Manager Associates](https://git.k8s.io/sig-release/release-managers.md#associates) |
```

Comment:
Let's remove the table entry from here to make it clear that Branch Management is no longer part of the Release Team.

Instead, add this note below the table:
```
Review the [Release Managers page](/release-managers.md) for up-to-date contact information on Release Engineering personnel.

The schedule for all patch releases can be found [here](/releases/patch-releases.md). It will be updated to include 1.16, once the 1.16 release cycle concludes.
```

Reply:
sounds good!